class-instant-articles-post.php

```diff
@@ -689,8 +689,11 @@ public function to_instant_article() {
 			Image::setDefaultCommentEnabled( $settings_publishing[ 'comments_on_media' ] );
 			Video::setDefaultCommentEnabled( $settings_publishing[ 'comments_on_media' ] );
 		}
-
-		$transformer->transformString( $this->instant_article, $this->get_the_content(), get_option( 'blog_charset' ) );
+		
+		$the_content = $this->get_the_content();
+		if ( '' != $the_content ) {
```

Comment:
@vkama i'm sorry, i wish i understand how to use the suggested `Type:isTextEmpty()` 😰 Would you mind show me sample?